uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -633,6 +633,11 @@ define([
       this.initAfterContactSelection = function () {
         var self = this;
 
+        //when manager deselects contact it is called without a selected contact_id
+        if (!self.request.contact_id) {
+          return $q.reject("The contact id or period id was not set");
```

Comment:
Why is the error message mentioning the period id if the check is on `self.request.contact_id`?